### PR TITLE
Update docstrings: remove redundant ones and tweak spelling on others.

### PR DESCRIPTION
**Pull request by mikez**
- Removed redundant docstrings.
- Made other docstrings consistent.

Docstrings in tests are very uncommon. Usually the convention is to make the name of the method self-explanatory and add comments if needed.

### CODE REVIEW

**Comment by mikez** [comment]
@AlexanderWillner Can you tell Pydocstyle not to check the tests directory?

File: things/database.py [hunk 1]
```diff
@@ -626,7 +626,7 @@ def make_filter(column, value):
 
 def make_date_filter(date_column, offset):
     """
-    Return an SQL filter to limit the date range of the SQL query.
+    Return a SQL filter to limit the date range of the SQL query.
```
Comment (AlexanderWillner) [inline things/database.py:629]: Actually, both is correct ;)

**Review by AlexanderWillner — APPROVED** [review]

**Comment by AlexanderWillner** [comment]
> @AlexanderWillner Can you tell Pydocstyle not to check the tests directory?

Sure.

**Comment by AlexanderWillner** [comment]
`make them human-legible` -> well, if removing them is making them human-ledigble, then lgtm (looks good to me) ;)

File: things/database.py (hunk 1 above)
Comment (mikez) [reply things/database.py:629]: haha, you're right! It depends on how you pronounce it, I see it now. :-)
I think I compared "a SQL filter" and "an SQL filter" on Google and the former had more hits.

**Comment by mikez** [comment]
> `make them human-legible` -> well, if removing them is making them human-ledigble, then lgtm (looks good to me) ;)

hahahaha, good point. 😂 
Let me change that title.

**Pull request merged**